src/command/repo/get.c

```diff
@@ -46,13 +49,96 @@ storageGetProcess(IoWrite *destination)
 
             if (repoCipherType != cipherTypeNone)
             {
+                /*******************************************************************************************************************
+                Repository encryption design:
+                REPO / (repoCipherPass)
+                     / archive   / (repoCipherPass)
+                     / archive   / stanza / (archiveCipherPass)
+                     / backup    / (repoCipherPass)
+                     / backup    / stanza / (backupCipherPass)
+                     / backup    / stanza / set / (manifestCipherPass)
+                     / backup    / stanza / backup.history / (manifestCipherPass)
```

Comment:
Isn't the backup.history dir encrypted with backupCipherPass?

Reply:
Indeed, wrong comment you're right (as always).
That's what is done in the code, apply backupCipherPass.
I'll update the comment and look at coverage tests.

Thanks !

Reply:
Haha, well, I've been doing this for a while. I saw the code was correct so the comment was just a typo.

Reply:
Modified, thanks ;-)